src/account-info/AccountDetailsScreen.js

```diff
@@ -52,6 +59,6 @@ class AccountDetailsScreen extends PureComponent<Props> {
   }
 }
 
-export default connectFlowFixMe((state: GlobalState, props) => ({
+export default connect((state, props): SelectorProps => ({
```

Comment:
This line will change slightly after #3842, the fix to #3768. Similarly elsewhere.

As is, I think this `SelectorProps` annotation may not actually have any effect! I'm no longer sure what problem I believed it worked around, or why, when I added that pattern last year in #3450. But the new form:
```
export default connect<SelectorProps, _, _>((state, props) => ({
```
successfully fixes #3768 (works around the underlying Flow issue), so that'll be good.